int64 id 
geometry_msgs/Pose pose
geometry_msgs/Twist velocity
float64 heading
